augly/image/functional.py

```diff
@@ -1692,6 +1722,7 @@ def pad_square(
     image = imutils.validate_and_load_image(image)
 
     func_kwargs = imutils.get_func_kwargs(metadata, locals())
+    src_mode = image.mode
```

Comment:
Unresolving because I still don't see the change in here! It should be on what is now line 1744.

Reply:
Pushed the fix for this one now :)

Reply:
+1 on my previous comment -- you can undo these changes because as long as `pad` handles the modes correctly doing this in `pad_square` is redundant (cc: @zpapakipos)